src/identity.ts

```diff
@@ -2,175 +2,126 @@
  * See LICENSE.md in the project root.
```

Comment:
Just 2 general thoughts regarding this file: 
- small thing, you've changed strings from this notation '' to "" 😅 
- in some cases (e.g. `IdentityOptions`) you've introduced duplicate declarations, both in `identity.d.ts` and `identity.ts` file - is there any reason for that?

Reply:
Sorry about the first, I guess lack of ts-lint and/or prettier config made " my formatting default. Will ofc fix this!
Regarding duplicates in identity.d.ts & identity.ts those are expected since identity.d.ts is now generated from identity.ts then all exported members should be present in both files. (Unless I misunderstood something).